index.html

```diff
@@ -19,7 +19,7 @@
             <a class="landing-a" href="#">Cart Management</a>
         </li>
         <li class="landing-li">
-            <a class="landing-a" href="#">Wishlist Management</a>
+            <a class="landing-a" href="/HTML and CSS/Wishlist/wishlist.html">Wishlist Management</a>
```

Comment:
- good classes names
- looks good

Reply:
index.html is just have links of pages like wishlist, home page and will be deleting that page after some time thats why I dont focus on class name but will fix that now thanks for pointing it out

Reply:
Hey @harshitpaliwal95 gave good class name please have a look